std/child_process.zig

```diff
@@ -768,3 +770,47 @@ pub fn createWindowsEnvBlock(allocator: *mem.Allocator, env_map: *const BufMap)
     i += 1;
     return allocator.shrink(result, i);
 }
+
+// TODO: does this already exist somewhere?
+fn Chain(comptime T: type, comptime U: type) type {
```

Comment:
Note that this (and the `OneItemIterator`) should either be moved somewhere else before merging, or they need to be removed if there is something else in the standard library that already provides this.

I'm not familiar enough with the standard library to know if these already exist somewhere in some form.